x-pack/plugins/apm/common/service_health_status.ts

```diff
@@ -7,7 +7,7 @@
 import { i18n } from '@kbn/i18n';
 import { ANOMALY_SEVERITY } from '../../ml/common';
 
-import { EuiTheme } from '../../../legacy/common/eui_styled_components';
+import { EuiTheme } from '../../xpack_legacy/common';
```

Comment:
We can actually import this from the observability plugin but it's in `public` there so we'll have to move that around in a separate PR.